compose/README.md

```diff
@@ -0,0 +1,61 @@
+![Project in Android Studio](screenshots/studio.png)
+
+Composite build of [Compose-jb sources](https://github.com/JetBrains/androidx)
+
+## Download submodules after downloading the main project:
+```
+git submodule update --init
+```
+Set this property to always update submodules on git checkout/pull/reset:
+```
+git config --global submodule.recurse true
```

Comment:
Shall it indeed be global, cannot we make it project local?

Reply:
Unfortunately I haven't found a way to

Reply:
I meant just `git config submodule.recurse true`

Reply:
It only changes the local config for the local repo, if I checked correctly.